docs/usage/self-hosted-configuration.md

```diff
@@ -160,34 +160,9 @@ For this to work, `docker` needs to be installed and the Docker socket available
 Now Renovate uses `docker run` to create containers like Node.js or Python to run tools in as-needed.
 
 Additionally, when Renovate is run inside a container built using [`containerbase`](https://github.com/containerbase), such as the official Renovate images on Docker Hub, then `binarySource=install` can be used.
-This mode means that Renovate will dynamically install the version of tools available, if supported.
-
-Supported tools for dynamic install are:
-
-- `bundler`
-- `cargo`
-- `composer`
-- `dotnet`
-- `flux`
-- `golang`
-- `gradle-wrapper`
-- `helm`
-- `jb`
-- `jsonnet-bundler`
-- `lerna`
-- `mix`
-- `node`
-- `npm`
-- `pip_requirements`
-- `pip-compile`
-- `pipenv`
-- `pnpm`
-- `poetry`
-- `python`
-- `rust`
-- `yarn`
-
-If all projects are managed by Hermit, you can tell Renovate to use the tooling versions specified in each project via Hermit by setting `binarySource=hermit`.
+This mode means that Renovate will dynamically install the version of tools available, if supported. // I think we now support install-mode for all tools???
```

Comment:
Can you suggest a change to this line, I don't know how to change it now we support `install-mode` for all tools? 😕